### ISSUE TITLE
Wrong upgrading suggestion

### ISSUE THREAD

**Issue by haithngn** [JanDeDobbeleer/oh-my-posh#4697]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

## What happened?

The Posh keeps suggesting upgrading even when the latest version is installed.

```
~ ❯ source ~/.oh-my-posh.nu  
A new release of Oh My Posh is available: v19.5.2 → v19.6.0
To upgrade, run: 'brew update && brew upgrade oh-my-posh'
```

Actually,
```
~ ❯ oh-my-posh version                                                                                                                                     10:40:58 
19.11.3
```

### Theme

any

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

nu

**Comment by JanDeDobbeleer** [comment]
@haithngn that's weird as the `v19.5.2` version number comes from inside the executable, just like when using oh-my-posh version. What does the content of `~/.oh-my-posh.nu` look like? I'm quite certain you have 2 oh-my-posh executables on your machine somehow.

**Comment by haithngn** [comment]
@JanDeDobbeleer, you are right there is some weird in that file content.

I can resolve this issue now it seems like a typing mistake.

```
$env.PROMPT_COMMAND_RIGHT = { ||
    ^"/opt/homebrew/bin/oh-my-posh" print right $"--config=($env.POSH_THEME)" --shell=nu $"--shell-version=($env.POSH_SHELL_VERSION)" $"--execution-time=(posh_cmd_duration)" $"--status=($env.LAST_EXIT_CODE)" $"--terminal-width=(posh_width)"
}

if "false" == "true" {
    $env.TRANSIENT_PROMPT_COMMAND = { ||
        ^"/opt/homebrew/bin/oh-my-posh" print transient $"--config=($env.POSH_THEME)" --shell=nu $"--shell-version=($env.POSH_SHELL_VERSION)" $"--execution-time=(posh_cmd_duration)" $"--status=($env.LAST_EXIT_CODE)" $"--terminal-width=(posh_width)"
    }
}

if "true" == "true" {
    echo "
A new release of Oh My Posh is available: v19.5.2 → v19.6.0
To upgrade, run: 'brew update && brew upgrade oh-my-posh'
"
}
```

**Comment by JanDeDobbeleer** [comment]
@haithngn BTW, when using nu, that file is only updated after initialization of a new shell (and sometimes it fails to get created, which could be a timing issue or weird internal nu thing). So if you upgrade oh-my-posh but not restart the shell, this will be incorrect.
